core/invoker/build.gradle

```diff
@@ -13,6 +13,7 @@ repositories {
 dependencies {
     compile "org.scala-lang:scala-library:${gradle.scala.version}"
     compile project(':common:scala')
+    compile 'org.apache.curator:curator-recipes:2.12.0'
```

Comment:
Any reason not to use the latest curator 4.0?

Reply:
We deploy zookeeper 3.4.  In theory, curator 4.0 supports both 3.4 and 3.5, but it didn't work for me (runtime errors due to unsupported operations).  I tried the gradle exclusion incantation from https://curator.apache.org/zk-compatibility.html to try to get 3.4 compatability, but it didn't help.

So, I stuck with 2.12.0 which is the most recent release of the zookeeper 3.4 compatible line of curator.